quinn-proto/src/connection/spaces.rs

```diff
@@ -436,6 +433,72 @@ impl SendableFrames {
     }
 }
 
+#[derive(Debug)]
+pub(crate) struct PendingAcks {
+    permit_ack_only: bool,
+    pending_acks: ArrayRangeSet,
```

Comment:
Maybe rename this to `ranges` now? Feels a bit stuttery otherwise.

Reply:
done